### PR TITLE
Edits to CONTRIBUTING.md for spelling, grammar, and clarity.

### PR DESCRIPTION
**Pull request by TimSchumann**
- Changed all text based instances of LVGL to uppercase (Only instances referring to LVGL the project, not instances referring to the ‘lvgl’ directory structure in the repository, or in URL links)
- Standardized bulleted/numbered list capitalization and punctuation
- Fixed several typos and spelling errors
- Changed minor grammatical errors and structure for clarity.
- LINE 33 - Removed extra empty line.
- LINE 156 - Added a link to the lvgl/examples/widgets directory for clarity
- LINE 207/211 - Fixed a tabs/spaces indentation issue on nested bulleted list
- LINE 152 - Need clarification on a possible typo.  Not sure if this was just a typo of the word ‘index’ or if there is something I’m missing here.

Also, there are several broken links to other places in the documentation, I don't know enough about how all of these files are linked but I have notes on which links/lines are broken. I just want to make sure they're not fixable in some programmatic way  before I inline anything. I'll circle back to that on Monday.

### CODE REVIEW

File: docs/CONTRIBUTING.md [hunk 1]
```diff
@@ -168,7 +167,7 @@ As you use LVGL you might find bugs. Before reporting them be sure to check the
 
 If it really seems like a bug feel free to open an [issue on GitHub](https://github.com/lvgl/lvgl/issues). 
 
-When filing the issue be sure to fill the template. It helps a lot to find the root of the problems and helps to avoid a lot of questions.
+When filing the issue be sure to fill the template. It helps find the root of the problem, avoid additional questions, or extended exchanges with other developers.
```
Comment (embeddedt) [inline docs/CONTRIBUTING.md]: The second sentence still feels a bit awkward... perhaps something like this?

```suggestion
When filing the issue be sure to fill out the template. It helps find the root of the problem while avoiding extensive questions and exchanges with other developers.
```

File: docs/CONTRIBUTING.md [hunk 2]
```diff
@@ -97,18 +96,18 @@ For example in case of StackOwerflow a notice like this can be used:
...
 
 #### Use MIT licensed code
-As LVGL is also MIT licensed other MIT licensed code can be integrated without issues.
-The MIT license requests a copyright notice be added to the derived work. So you need to copy the original work's license file or it's text to the code you want to add.
+As LVGL is MIT licensed, other MIT licensed code can be integrated without issues.
+The MIT license requires a copyright notice be added to the derived work. Any derivative work based on MIT licensed code must copy the original work's license file or text.
 
 #### Use GPL licensed code
-As GPL license is not compatible with MIT license so LVGL can not accept GPL licensed code. 
+The GPL license is not compatible with the MIT license, as such LVGL can not accept GPL licensed code.
```
Comment (embeddedt) [inline docs/CONTRIBUTING.md]: This is technically a comma splice. How about this?

```suggestion
The GPL license is not compatible with the MIT license. Therefore, LVGL can not accept GPL licensed code. 
```

**Review by embeddedt — APPROVED** [review]
Thank you! This is a great contribution.

I'm a bit of a grammar fanatic so I found two spots which could be improved a bit more, but feel free to ignore my suggestions. :wink:

**Comment by TimSchumann** [comment]
Great changes @embeddedt. I'm new to the C language and using LVGL for a project, contributing to the docs is a great way to trick my monkey brain into reading things I need to learn, so possibly look forward to more from me in the future. Also, I tagged you in a forum post, could use some help there.

**Comment by embeddedt** [comment]
Thanks! I've just merged this.

I commented on your forum post.

**Pull request merged**